watchtower-plugin/src/net/http.rs

```diff
@@ -154,6 +155,24 @@ pub async fn send_appointment(
     }
 }
 
+/// Handles a ping request to the tower.
+pub async fn ping(server_url: &str) -> Result<(), Box<dyn std::error::Error>> {
+    let client = Client::new();
+    let response = client.get(&format!("{}/{}", server_url, Endpoint::Ping)).send().await?;
+
+    match response.status().is_success() {
```

Comment:
This match could be simplified into an `if else` statement.

Reply:
sure! Thanksss